script/lint.js

```diff
@@ -13,6 +13,8 @@ const SOURCE_ROOT = path.normalize(path.dirname(__dirname));
 const DEPOT_TOOLS = path.resolve(SOURCE_ROOT, '..', 'third_party', 'depot_tools');
 
 const IGNORELIST = new Set([
+  ['filenames.libcxx.gni'],
+  ['filenames.libcxxabi.gni'],
```

Comment:
These shouldn't need to be ignored (E.g. filenames.auto.gni doesn't appear to be ignored)

Reply:
Silly thing, but the reason I added them is that our linting changes the licenses array, an array with one value from this:
```
libcxxabi_licenses = [
  "//buildtools/third_party/libc++abi/trunk/LICENSE.TXT",
]
```
to this:
```
libcxxabi_licenses = [
  "//buildtools/third_party/libc++abi/trunk/LICENSE.TXT",
]
```

This change then causes the `gn --check` to fail, because they don't match. I added them to the lint ignore for the simplest fix, but there may be a better way to handle that.

Reply:
Oh sure, just update the generation thing to generate it so it looks like that 😄

Reply:
Ahaha, I'm an idiot, it's literally just changing a template string - thanks, changing :slightly_smiling_face: